skins/oasis/modules/BodyController.class.php

```diff
@@ -239,7 +239,7 @@ public function getRailModuleList() {
 			return [ ];
 		}
 
-		$railModuleList[1440] = [ 'Ad', 'Index', [ 'slotName' => 'TOP_RIGHT_BOXAD' ] ];
+		$railModuleList[1440] = [ 'Ad', 'Index', [ 'slotName' => 'TOP_BOXAD' ] ];
```

Comment:
Cache is going to hurt us 🙈

Reply:
Inject both slots, for a while. It shouldn't hurt us. I believe

Reply:
Fixed.